helix-view/src/editor.rs

```diff
@@ -289,6 +297,9 @@ pub struct Config {
     pub default_line_ending: LineEndingConfig,
     /// Enables smart tab
     pub smart_tab: Option<SmartTabConfig>,
+    #[serde(default)]
+    /// What the diff gutter should diff against
+    pub diff_source: DiffSource,
```

Comment:
I think this should be just  `editor.diff-source` since the diff is also used for the `reset-diff-change` command and `]g` and also for a side by side diff in the future so it doesn't just pertain to the gutter

Reply:
this actually is just `editor.diff-source`, i guess the comment is a bit misleading. i'll clean up the docs etc once the feature itself is more finalized.

Reply:
ah sorry github collapsed the code so that it looked like it was a gutter config field and the comment reinforced that idea :sweat_smile: 

![image](https://github.com/helix-editor/helix/assets/61850714/72aa2e93-e194-4412-917a-56b58306975c)